package com.sfsymbols.data.dualtone

import androidx.compose.ui.graphics.vector.ImageVector
import com.sfsymbols.data.SfSymbols
import com.sfsymbols.data.addSfPath
import com.sfsymbols.data.sfIcon

/**
 * SF Symbol: SFOvalRighthalfFilled (dualtone)
 * Viewport: 32.168 x 24.0137
 */
public val SfSymbols.Dualtone.SFOvalRighthalfFilled: ImageVector
    get() {
        if (_sFOvalRighthalfFilled != null) {
            return _sFOvalRighthalfFilled!!
        }
        _sFOvalRighthalfFilled = sfIcon(
            name = "Dualtone.SFOvalRighthalfFilled",
            viewportWidth = 32.168f,
            viewportHeight = 24.0137f
        ) {
            addSfPath("M31.8066 12.002C31.8066 5.0293 25.2148 0 15.9082 0C6.5918 0 0 5.0293 0 12.002C0 18.9746 6.5918 24.0137 15.9082 24.0137C25.2148 24.0137 31.8066 18.9746 31.8066 12.002ZM15.9082 22.2754C7.60742 22.2754 1.72852 17.959 1.72852 12.002C1.72852 6.05469 7.60742 1.73828 15.9082 1.73828Z", fillAlpha = 0.85f)
        }
        return _sFOvalRighthalfFilled!!
    }

private var _sFOvalRighthalfFilled: ImageVector? = null
